### PR TITLE
created jarvis profile, attempt to merge from feature to master

### PR DESCRIPTION
**Pull request by Hunain-Khalid**

### CODE REVIEW

File: profile/profile.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Preferred name (Characters only)
+name: "Hunain Khalid"
+# Your GitHub repo root URL
+github_repo_root_url: "https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid"
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: "Hello, I am Hunain Khalid a software developer with a passion for machine learning (ML) and artifical intelligence (AI).
```
Comment (trevor-jrvs) [inline profile/profile.yaml:8]: I would rewrite this intro. I will message you some sample profiles.
-No need to introduce yourself by name, your name is on top of the page.
-Include more details about your technical skills and experience
-Ensure that grammar is correct: consider running your profile through grammarly's checking service

File: profile/profile.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    description: "WIP"
+  - name: "Hadoop"
+    git_url: "/hadoop"
+    description: "WIP"
+  - name: "Spark"
+    git_url: "/spark"
+    description: "WIP"
+  - name: "Cloud/DevOps"
+    git_url: "/cloud_devops"
+    description: "WIP"
```
Comment (trevor-jrvs) [inline profile/profile.yaml:64]: remove unstarted projects, fill in details on current WIP project.

File: profile/profile.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+#List of professional experience
+#The job description allows hiring managers to understand your job and the technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+#The description must be more than 25 words. If your experience is not software related, please focus on soft skills and achievements (e.g. team collaboration, problem-solving, improvements, communication).
+professional_experience:
+  #Write your Jarvis role as a professional experience rather than training.
+  - title: "Software Developer"
+    company: "Jarvis Consultancy Group"
+    duration: "2021-present"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Created six projects, all in an agile environment, closely monitored by senior developers
```
Comment (trevor-jrvs) [inline profile/profile.yaml:89]: You haven't completed six projects yet, have you?

**Review by trevor-jrvs — CHANGES_REQUESTED** [review]
rewrite intro, fix notes, and resubmit

**Comment by Hunain-Khalid** [comment]
closing PR

**Pull request closed without merging**